### PR TITLE
add Cut transformer

### PR DESCRIPTION
**Pull request by omercnet**
## Status
Ready

## Description
A Cut filter that (unlike Split) can return multiple fields, similar to the unix cut command

## Screenshots
image

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
None

## Additional changes
Describe additional changes done, for example adding a function to common server.

fixes https://github.com/demisto/etc/issues/15192

### CODE REVIEW

File: Scripts/script-Cut.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+commonfields:
+  id: cae96081-c034-46c1-81f6-b4d0a52dc51c
```
Comment (asafshen) [inline Scripts/script-Cut.yml]: change ID to be same as name 
also set version to `-1`

File: Scripts/script-Cut.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  data = value.split(delim)
+  fields = [int(_) for _ in fields.split(',')]
+
+  demisto.results(delim.join([str(data[i-1]) for i in fields]))
+type: python
+tags:
+- transformer
+- string
+comment: Cut a string by delimiter and return specific fields
```
Comment (asafshen) [inline Scripts/script-Cut.yml]: lets consult with Alex about this comment.

I would give an example (e.g `Cut value=<val> delimiter=<del> ... => <result-example>`)

File: Scripts/script-Cut.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+commonfields:
+  id: cae96081-c034-46c1-81f6-b4d0a52dc51c
+  version: 38
+name: Cut
+script: |-
+  args = demisto.args()
+
+  value = args.get('value')
+  fields = args.get('fields')
+  delim = args.get('delimiter')
```
Comment (asafshen) [inline Scripts/script-Cut.yml]: I would verify input correctness:
- types (`value` and `delimiter` should be string, `fields` should be csv numbers)
- indexes in range

File: Scripts/script-Cut.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+args:
+- name: value
+  required: true
+  description: Value to split
+- name: delimiter
+  required: true
+  description: Delimiter to cut the string by
+- name: fields
+  required: true
+  description: 'Field numbers (e.g.: 1,5,7)'
```
Comment (asafshen) [inline Scripts/script-Cut.yml]: `Comma separated indexes to include in the result`?

File: Scripts/script-Cut.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+comment: Cut a string by delimiter and return specific fields
+enabled: true
+args:
+- name: value
+  required: true
+  description: Value to split
+- name: delimiter
+  required: true
+  description: Delimiter to cut the string by
+- name: fields
```
Comment (asafshen) [inline Scripts/script-Cut.yml]: I would name `indices` and not `fields`

Comment (omercnet) [reply Scripts/script-Cut.yml]: I'm borrowing the `cut` unix command syntax and parameters that people might be familiar with

File: Scripts/script-Cut.yml (hunk 4 above)
Comment (omercnet) [reply Scripts/script-Cut.yml]: fixed

**Comment by anara123** [comment]
@omercnet can you also add a test?
https://github.com/demisto/internal-content/tree/master/documentation/tests

**Comment by anara123** [comment]
@omercnet on the second thought, forget about the test. We will take care of it

**Pull request merged**